bower.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "main": [
     "dist/impac-angular.js",
-    "src/impac-angular.less"
+    "dist/impac-angular.less"
```

Comment:
@ouranos I think this is fine, but just confirming that parent apps less overrides, etc, will still function as they were when pointing to `src/impac-angular.less`? What's different about the new concatenated dist less file that allows us to do this now?